CMS/settings/dev.py

```diff
@@ -3,16 +3,14 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-# SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = '3z@sut7t&=hkx9*+b-h=&2$_ajcqvqrfcc0mc@*)y6c5v#6qsz'
-
 # SECURITY WARNING: define the correct hosts in production!
-ALLOWED_HOSTS = ['*'] 
+ALLOWED_HOSTS = ['*']
 
 EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
 
+DEFAULT_FILE_STORAGE = 'django.core.files.storage.FileSystemStorage'
 
-TEST_ENV = 'dev'
+SECRET_KEY = '3z@sut7t&=hkx9*+b-h=&2$_ajcqvqrfcc0mc@*)y6c5v#6qsz'
```

Comment:
Don't you want to have the associated comment above this line instead of line 6?

Reply:
Done!
https://github.com/office-for-students/wagtail-CMS/blob/rob-fixes/CMS/settings/dev.py#L13